conf/carbon.properties.template

```diff
@@ -42,6 +42,9 @@ carbon.enableXXHash=true
 #carbon.max.level.cache.size=-1
 #enable prefetch of data during merge sort while reading data from sort temp files in data loading
 #carbon.merge.sort.prefetch=true
+######## Alter Partition Configuration ########
+#Number of cores to be used while alter partition
+carbon.number.of.cores.while.altPartition=2
```

Comment:
Yes, it will be used when take action of multiple segments in parallel. this configuration will allow user to set the threads according to their hardware.
Sure, I will make the change.